vendor/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -293,9 +292,6 @@ var newETCD3Client = func(c storagebackend.TransportConfig) (*clientv3.Client, e
 		Endpoints:            c.ServerList,
 		TLS:                  tlsConfig,
 		Logger:               etcd3ClientLogger,
-
-		MaxCallRecvMsgSize: math.MaxInt32,
-		MaxCallSendMsgSize: math.MaxInt32,
```

Comment:
chore: since we are swapping the storage and removing etcd, we should also get rid of vendoring